lms/static/scripts/frontend_apps/utils/api.ts

```diff
@@ -111,9 +111,7 @@ export async function apiCall<Result = unknown>(
     headers['Content-Type'] = 'application/json; charset=UTF-8';
   }
 
-  const queryString = recordToSearchParams(params ?? {}).toString();
-  const query = queryString.length > 0 ? `?${queryString}` : '';
-
+  const query = recordToQueryStringFragment(params ?? {});
```

Comment:
Since I found myself needed this logic in a third location, I extracted it to a new helper function.